docs-source/content/userguide/managing-domains/_index.md

```diff
@@ -132,3 +134,65 @@ The operator let's you initiate scaling of clusters in various ways:
 * [Using the operator's REST APIs]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling#calling-the-operators-rest-scale-api" >}})
 * [Using WLDF policies]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling#using-a-wldf-policy-rule-and-script-action-to-call-the-operators-rest-scale-api" >}})
 * [Using a Prometheus action]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling#using-a-prometheus-alert-action-to-call-the-operators-rest-scale-api" >}})
+
+### Log files
+
+The operator can automatically override WebLogic domain, server, and introspector `.log` and `.out` locations.
+This occurs if the Domain `logHomeEnabled` field is explicitly set to `true`, or if `logHomeEnabled` isn't set
+and `domainHomeSourceType` is set to `PersistentVolume`.  When overriding, the log location will be the location specified by the `logHome` setting.
+
+If you want to fine tune the `.log` and `.out` rotation behavior for WebLogic domains and servers, then
+you can update the related `Log MBean` in your WebLogic configuration. Alternatively, for WebLogic
+servers, you can set corresponding system properties in `JAVA_OPTIONS`:
```

Comment:
WebLogic servers -> WebLogic Servers (Because WebLogic Server is a legal product name, it should always be capitalized)
In your previous sentence, "...rotation behavior for WebLogic domains and servers," I recommend that you re-write it as "...rotation behavior for WebLogic Servers and domains."

Reply:
fixed